README.md

```diff
@@ -19,6 +19,39 @@ When security vulnerabilities arise, CBL-Mariner supports both a package-based u
 
 Instructions for building CBL-Mariner may be found here: [Toolkit Documentation](./toolkit/README.md)
 
+## ISO
+You can try out CBL-Mariner with the following ISO: [x86_64 ISO](https://osrelease.download.prss.microsoft.com/pr/download/Mariner-1.0-x86_64.iso)
+
+Note: The ISO is intended for testing in virtual environments. Because it is intended for use in Azure, it contains intentionally limited hardware support.
+
+### ISO Quickstart with Hyper-V
+**Create VHD(X) Virtual Machine with Hyper-V**
+
+1. From Hyper-V Select _Action->New->Virtual Machine_.
+1. Provide a name for your VM and press _Next >_.
+1. Select _Generation 1_ (VHD) or _Generation 2_ (VHDX), then press _Next >_.
+1. Change Memory size if desired, then press _Next >_.
+1. Select a virtual switch, then press _Next >_.
+1. Select _Create a virtual hard disk_, choose a location for your VHD(X) and set your desired disk Size.  Then press _Next >_.
+1. Select _Install an operating system from a bootable image file_ and browse to your Demo ISO. 
+1. Press _Finish_.
+
+**[Gen2/VHDX Only] Additional Boot Configuration**
+
+1. Right click your virtual machine from Hyper-V Manager
+1. Select _Settings..._
+1. Select Security and under _Template:_ select _Microsoft UEFI Certificate Authority_.
+1. Select Firmware and adjust the boot order so DVD is first and Hard Drive is second.

```

Comment:
Is this step really only specific to gen 2 VMs? I'm not saying it's not, it just sounds weird.:)

Reply:
This is really a matter of defaults. Gen1 default boot order is CD, IDE, Network, Floppy. Gen2 default is DVD, Network Adapter, Hard Drive. So only Gen2's default boot order will need to be updated.

Reply:
Oooooooooh, thanks.:)